src/MultiOwnable.sol

```diff
@@ -82,6 +82,10 @@ contract MultiOwnable {
         return _getMultiOwnableStorage().isOwner[account];
     }
 
+    function isOwnerPublicKey(bytes32 x, bytes32 y) public view virtual returns (bool) {
+        return _getMultiOwnableStorage().isOwner[abi.encode(x, y)];
```

Comment:
we should probably then make `isOwnerBytes` internal and have `isOwnerPublicKey` and `isOwnerAddress` call to it

Reply:
The key ways to view owners of this contracts are as:
- address
- public key
But IMO there should also be a universal method of `isOwner()` which ignores the type of owner. thoughts?

Reply:
I guess I think it should be symmetric to our addOwner functions. Do we use isOwnerBytes anywhere in the code?

Reply:
We use `isOwnerBytes()` once as a requirement within `_addOwnerAtIndexNoCheck()`.  So, my question is do we want to restrict owners to specifically `address` or `bytes32[](2)`?  If yes, then I should remove the external view.

Reply:
> We use isOwnerBytes() once as a requirement within _addOwnerAtIndexNoCheck().
Right, so we can leave as an internal function 

> So, my question is do we want to restrict owners to specifically address or bytes32[](2)
Yes, we already do because we only have the two add owner functions

Reply:
though idk I guess I could be convinced it's nice to leave isOwnerBytes as a convenience method. You may not want to check the bytes length and use the others

Reply:
I don't think it's a big deal if we leave it.  What's the worst that could happen?